src/controllers/authentication/user.js

```diff
@@ -58,3 +52,37 @@ export const createUser = async (req, res) => {
     }));
   }
 };
+
+export const viewUser = async (req, res) => {
+  try {
+    const user = await User.findByPk(req.params.id, {
+      attributes: {
+        exclude: [
+          'password',
+          'createdAt',
+          'updatedAt',
+          'id',
+          'notification',
+          'role',
+        ],
+      },
+    });
+
+    if (user) {
+      return res.status(200).json(responseFormat({
+        status: 'success',
+        data: {
+          user,
+        },
+      }));
+    }
+
+    return res.status(404).json(errorResponseFormat({
+      message: 'This profile does not exist',
+    }));
+  } catch (error) {
+    return res.status(500).json(errorResponseFormat({
+      message: 'Something Went Wrong',
```

Comment:
You might want to return `status` here.

Reply:
the errorResponseFormat() has a default status of error, i only need to pass message into the function